.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: objective-c
-osx_image: xcode10
+osx_image: xcode10.2
 
 script:
-   - xcodebuild -scheme ELMaestro -sdk iphonesimulator clean test -destination 'OS=12.0,name=iPhone XR' CODE_SIGNING_REQUIRED=NO
-   - xcodebuild -scheme ELMaestro_static -sdk iphonesimulator clean build -destination 'OS=12.0,name=iPhone XR' CODE_SIGNING_REQUIRED=NO
+   - xcodebuild -scheme ELMaestro -sdk iphonesimulator clean test -destination 'OS=12.2,name=iPhone Xʀ' CODE_SIGNING_REQUIRED=NO
```

Comment:
is `Xʀ` different than `XR` ?

Reply:
yeah, surprised me too. take a look at this build log: https://travis-ci.org/Electrode-iOS/ELMaestro/builds/518835760#L941-L954